.github/workflows/docker.yml

```diff
@@ -59,4 +60,23 @@ jobs:
           # Docker compose can take some time to start
           max_attempts: 5
           timeout_minutes: 2
-          command: sleep 60 && linera --wallet docker/wallet.json --storage rocksdb:docker/linera.db sync-balance
\ No newline at end of file
+          command: sleep 60 && linera --wallet docker/wallet.json --storage rocksdb:docker/linera.db sync-balance
+      - name: Run benchmarks
+        run: |
+          ./scripts/bench.sh docker/wallet.json rocksdb:docker/linera.db > benchmark_results.txt
```

Comment:
Why not `results=$(./scripts/bench.sh docker/wallet.json rocksdb:docker/linera.db)` directly, rather than writing it to a file and later `cat`ting that file?